File: First Unique Character in a String.py
class Solution:
    def firstUniqChar(self, s: str) -> int:
        counts = [0]*26
        
        for c in s:
            counts[ord(c) - ord('a')] += 1
            
        for i in range(0, len(s)):
            if counts[ord(s[i]) - ord('a')] == 1:
                return i
            
        return -1
        

File: First Unique Character in a String.java
class Solution 
{
    public int firstUniqChar(String s) 
    {
        int[] Freqcount = new int[26];
        for(int i = 0 ; i<s.length(); i++)
        {
            char ch = s.charAt(i);
            Freqcount[ch-'a']++;
        }
        
        for(int i = 0 ; i<s.length(); i++){
            char ch = s.charAt(i);
            if(Freqcount[ch-'a'] == 1) return i;
        }
        
        return -1;
    }
}